main.go

```diff
@@ -60,6 +60,8 @@ var (
 	submissionRunnerThreads = flag.Int("submission-threads", 3, "Number of worker threads "+
 		"used by the SparkApplication submission runner.")
 	resyncInterval = flag.Int("resync-interval", 30, "Informer resync interval in seconds")
+	namespace = flag.String("namespace", "", "The Kubernetes namespace to manage. "+
+		"Will manage CRDs for the whole cluster if unset.")
```

Comment:
Changing `CRDs` to `custom resource objects of the managed CRD types` makes it clearer.